sources/classes/ConfigParser.cpp

```diff
@@ -0,0 +1,217 @@
+#include "../../includes/classes/ConfigParser.hpp"
+
+std::vector<std::string>	ConfigParser::_strServerBlock;
+std::vector<std::string>	ConfigParser::_strLocationBlock;
+
+static void	printStrConfig(std::vector<std::string> &strServerConfig)
+{
+	for (std::vector<std::string>::iterator bgn = strServerConfig.begin(); bgn < strServerConfig.end(); bgn++)
+		std::cout << *bgn << std::endl;
+}
+
+void	ConfigParser::parseConfigs(std::string fileName, ServerBlocks &serverConfigs)

```

Comment:
Finish implementing this. @fmouronh

Reply:
The ConfigParser class is still a wip and is currently not being compiled nor used. Please ignore everything in that file